xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -268,6 +268,114 @@ std::string CGUIWindowVideoNav::GetQuickpathName(const std::string& strPath) con
   }
 }
 
+SELECT_FIRST_UNWATCHED_ITEM CGUIWindowVideoNav::SelectFirstUnwatchedItem()
+{
+  if (m_vecItems->IsVideoDb())
+  {
+    NODE_TYPE nodeType = CVideoDatabaseDirectory::GetDirectoryChildType(m_vecItems->GetPath());
+
+    if (nodeType == NODE_TYPE_SEASONS || nodeType == NODE_TYPE_EPISODES)
+    {
+      int iValue = CSettings::Get().GetInt("videolibrary.tvshowsselectfirstunwatcheditem");
+      if (iValue >= SELECT_FIRST_UNWATCHED_ITEM::NEVER && iValue <= SELECT_FIRST_UNWATCHED_ITEM::ALWAYS)
```

Comment:
If I don't then NEVER conflicts with another value in CGUIControl::GUISCROLLVALUE